### PR TITLE
[MOBILE-1737] Release 8.0.0

### PR DESCRIPTION
**Pull request by marc-scig**
Something that we want to get in before actually releasing 8.0.0. The majority of the changes are typescript related, but there were a few additions and removals. I also fixed a typo in MessageView.

### CODE REVIEW

**Comment by marc-scig** [comment]
I meant to combine this with the release ticket, given the branch name, so this will need a re-review

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
 
+## Version 8.0.0 - July 8, 2020
+Major release rewritten in TypeScript and adding separate modules for location,
+HMS and Accengage features
+
+- Added urbanairship-location-react-native module
+- Added urbanairship-hms-react-native module
+- Added urbanairship-accengage-react-native module
+- Added full TypeScript coverage
+- Removed Flow
```
Comment (rlepinski) [inline CHANGELOG.md:11]: We also updated sdks

Comment (marc-scig) [reply CHANGELOG.md:11]: oh right

Comment (marc-scig) [reply CHANGELOG.md:11]: fixed

**Pull request merged**